Slim/Networking/Async/HTTP.pm

```diff
@@ -166,8 +206,9 @@ sub send_request {
 		$self->request->protocol( 'HTTP/1.0' );
 	}
 	
+	# the used class is NET::HTTP::Method which now supports HTTP 1.1
```

Comment:
What do you mean by "now supports 1.1"? That module is shipped with LMS. Would that support it?

Reply:
Yes, the module shipped with LMS supports 1.1, AFAIK